src/schema/sale/display.ts

```diff
@@ -0,0 +1,98 @@
+import moment from "moment"
+
+export const isLiveOpen = sale => {
+  const liveStart = moment(sale.live_start_at)
+  return (
+    sale.auction_state === "open" &&
+    (moment().isAfter(liveStart) || moment().isSame(liveStart))
+  )
+}
+
+const hasEnded = end_at => {
+  return end_at && end_at < moment()
+}
+
+export async function displayTimelyAt(
+  sale,
+  useUpdatedFormatting: boolean,
+  meBiddersLoader
```

Comment:
This function should take an object `displayTimelyAt({ sale, useUpdatedFormatting, ... })` so that we'll be able to safely remove the `useUpdatedFormatting` argument without breaking the function signature.